public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -70,6 +70,7 @@ import {
 import {
   focusOnFirstWrongFeature,
   initialFeatureValue,
+  validateFeatures,
```

Comment:
Can this be removed? similar to https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/504/files#r1222290407